src/Symfony/Component/DependencyInjection/Compiler/ReplaceAliasByActualDefinitionPass.php

```diff
@@ -122,4 +124,13 @@ private function updateArgumentReferences(array $arguments, $currentId, $newId)
 
         return $arguments;
     }
+
+    private function updateFactoryServiceReference($factoryService, $currentId, $newId)
```

Comment:
I understand now, the 2.7 version has more code, so looks fine to me as it will ease merging.